try-it.md

```diff
@@ -38,17 +36,16 @@ permalink: /try-it.html
 - Run as a user with passwordless sudo access
 - Minimum resource requirements for the host machine: 4C CPUs, 16 GB RAM memory.
 
-> warning "Warning"
-> The system can be running CentOS 7. However, note that there is an ongoing process to move to latest CentOS version. Therefore, in order to avoid future issues you might find, CentOS 8 is the preferred CentOS choice.
-
-### 1.2. Metal3-dev-env setup
+### 1.2. Setup
```

Comment:
This title is less descriptive than it used to be. I guess "metal3-dev-env" isn't very informative to someone brand new to this, but maybe we can come up with some descriptive term for what is being set up?

Reply:
Agreed. I couldn't come up with something else than the old title. Reverting to the original version.